src/pages/ListView.vue

```diff
@@ -162,15 +176,25 @@ export default {
     },
     onAll() {
       this.activeFilter = "all"
-      this.todos = [...this.$store.getters.getFilterTodos("all")]
+      this.$store.commit(RESET_LIMIT)
+      this.$store.commit(SET_FILTER, {
+        filter: "all",
```

Comment:
introduce constant for 'all', "com", "inc"

Instead of short form, use "complete", "incomplete"